app/controllers/events_controller.rb

```diff
@@ -7,13 +7,14 @@ def index
   def search
     @event_list = Event.all
 
-    if (params[:event])
+    if (params[:event] == "messages")
+      @family_messages = Family.where.not(message: [nil, ``]
```

Comment:
Missing the closing `)` Also the 2nd element in the array needs to be quotes `''` not backticks. But if it works with this that means it should work with just `Family.where.not(message: nil)`